src/test/lambda/wizards/uploadLambdaWizard.test.ts

```diff
@@ -65,4 +72,30 @@ describe('UploadLambdaWizard', function () {
             tester.confirmedDeploy.assertShow(3)
         })
     })
+
+    describe('invoked from template', function () {
+        let tempDir: string
+        let tempDirUri: vscode.Uri
+        let invokePath: vscode.Uri
+        beforeEach(async function () {
```

Comment:
There's a bit of duplicate code in this beforeEach(). Please try to avoid that in the future.

Reply:
@justinmk3 These are in different `describe` blocks so they have different beforeEach() methods. Are you suggesting to place them in the same scope so they can both use that code? Maybe a 'invoked from explorer' `describe` block that contains both of these tests?

Reply:
- there is no real reason for different describe() blocks in this file. all the it() cases could live in one shared describe()
- or, code duplication can be avoided by extracting to a common function